keep-ui/app/rules/CorrelationSidebar/CorrelationSidebarBody.tsx

```diff
@@ -67,6 +73,7 @@ export const CorrelationSidebarBody = ({
             ruleName: name,
             celQuery: formatQuery(query, "cel"),
             timeframeInSeconds,
+            grouping_criteria: alertsFound.length ? groupedAttributes : [],
```

Comment:
`grouping_criteria` won't be sent to the backend if there are no alerts found. This ternary is a simple check to avoid clearing the GroupBy `MultiSelect` whenever `alertsFound.length === 0`, if that makes sense.

Reply:
It makes sense but looks like it sends that also when it's found.